mobile/bindings.go

```diff
@@ -12,6 +12,23 @@ import (
 	"github.com/lightningnetwork/lnd/signal"
 )
 
+// LndStatus holds the current status of lnd.
+type LndStatus struct {
+	// lndStarted tells whether the lnd is started or not.
```

Comment:
it looks these fields could use a mutex, since it's modified and read concurrently